lib/aptible/cli/subcommands/backup.rb

```diff
@@ -23,7 +23,8 @@ def self.included(thor)
               handle = options[:handle]
               unless handle
                 ts_suffix = backup.created_at.getgm.strftime '%Y-%m-%d-%H-%M-%S'
-                handle = "#{backup.database.handle}-at-#{ts_suffix}"
+                handle =
+                  "#{backup.database_with_deleted.handle}-at-#{ts_suffix}"
```

Comment:
```suggestion
                  "#{backup.database_handle}-at-#{ts_suffix}"
```

You should be able to just get the handle right off the backup now:
https://github.com/aptible/deploy-api/blob/bd1bab859462ad3a4ef98b41de01b5dd5108273d/app/decorators/backup_decorator.rb#L4

Reply:
What do I need to do to get my local CLI to recognize your change? I'm getting 
```
NoMethodError: undefined method `database_handle' for #<Aptible::Api::Backup:0x3ff969532828 @root="https://api-steve.aptible-sandbox.com" @href="https://api-steve.aptible-sandbox.com/backups/1" @loaded=true @namespace="Aptible::Api" ...>
  /Library/Ruby/Gems/2.6.0/gems/aptible-resource-1.1.0/lib/hyper_resource.rb:161:in `method_missing'
  /Users/steve/code/aptible-cli/lib/aptible/cli/subcommands/backup.rb:26:in `block (2 levels) in included'
  /Library/Ruby/Gems/2.6.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
  /Library/Ruby/Gems/2.6.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
  /Library/Ruby/Gems/2.6.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
  /Library/Ruby/Gems/2.6.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
  bin/aptible:11:in `<top (required)>'
```

Reply:
ahh, nevermind i didn't add it to the ruby client. we should just stay with yours as is.